Automattic/jetpack
# Jetpack Search Sync: Add French-Language Taxonomies

#### Changes proposed in this Pull Request:

- Add slugs "metier" and "profession" as they are French-language equivalents of "Profession." 
- Add slugs for French "fonction" for English "Function" and "sujet" for English "Theme"
- Add slugs with "question_" prefix to ensure they work with specific custom post types


*
- Adds French language equivalents for generic Jetpack Search slugs/taxonomies

#### Other information:

- [N/A] Have you written new tests for your changes, if applicable?
- [N/A] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:
- Add custom Jetpack Search taxonomies for "metier," "profession," "fonction," and "sujet" to site using Jetpack Search
- Search site using Jetpack Search
- Confirm that the custom taxonomies are available as filters for search results
